### PR TITLE
Update campaigns docs to include screenshots

### PR DESCRIPTION
**Pull request by mrnugget**
Just another small improvement to the campaign docs:

* Screenshots on the index page to immediately show users what campaigns
  are capable of
* Screenshot on the "Creating a manual campaign" to show of new,
  enhanced tracking capabilities
* Screenshots and improved explanation on "Updating campaigns"

### CODE REVIEW

File: doc/user/campaigns/index.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ Campaigns are part of [Sourcegraph code change management](https://about.sourceg
 
 You provide the code to make the change and Campaigns provide the plumbing to turn it into a large-scale code change campaign and monitor its progress.
 
+<div style="max-width: 500px;" class="mx-auto">
```
Comment (unknwon) [inline doc/user/campaigns/index.md:11]: qq: Is this `class="mx-auto"` necessary? Or did you just copy from another place in our docs?

Comment (mrnugget) [reply doc/user/campaigns/index.md:11]: `mx-auto` makes it aligned to the center. Without it, the image would be on the left.

**Comment by mrnugget** [comment]
@unknwon Can you take a look again? I've moved the images to Google Cloud.

**Comment by mrnugget** [comment]
I want to have this merged before others begin user testing. If there are any "changes requested", I'll follow up.

**Comment by unknwon** [comment]
Sorry about the delay, content LGTM, was just having concerns about adding more images to our git repo, but you've resolved it by uploading to GCS.

**Pull request merged**